karate-netty/README.md

```diff
@@ -476,6 +476,51 @@ You can use these in the "request matcher" described above. This is how you can
 
 > The [`pathParams`](#pathparams) is a special case. For each request, it will be initialized only if, and after you have used [`pathMatches()`](#pathmatches). In other words you have to call `pathMatches()` first - typically in the "request matcher" and then you will be able to unpack URL parameters in the `Scenario` body.
 
+##Scenario selection
+
+When multiple scenarios match request, scenario match score is used to pick first with highest score. If multiple scenarios have the same score, then the first one is picked. Scenario match score is based on (from most to least important):
+
+- `path` match score - using JAX-RS  
+        Number of literal characters
+        Number of path params without regex (i.e. regex missing)
+        Number of path params with explicit regex
+- `method` match
+- `query` parameter match(es)
+- `header` match(es) (this includes Accept and Content-Type functions)
```

Comment:
I think this part is from the old approach which is not applicable any more, right ?

Reply:
Removed the documentation from old approach.